test/Orchard.Tests/Orchard.Tests.csproj

```diff
@@ -30,4 +36,8 @@
     <PackageReference Include="xunit" Version="2.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```

Comment:
I already removed it but VS2017 re-adds it when the Test Explorer finds unit tests.
Oh, just find [this issue](https://github.com/nunit/nunit-vs-adapter/issues/51), i will take a look this night.
But you're right, i'll remove it ;)